src/lib/versionSelect.js

```diff
@@ -0,0 +1,38 @@
+const { prompt } = require('enquirer');
+const promptOpts = require('./prompts');
+const request = require('request-promise-native');
```

Comment:
Could we use [`node-fetch`](https://www.npmjs.com/package/node-fetch) here instead?

Reply:
Should I update this for all the commands as well?

Reply:
Nah, not in this PR at least! I believe we already have a ticket for refactoring the other usages

Reply:
Agreed that we should definitely move this to `node-fetch` so it's consistent with all of our OSS projects.

Reply:
Currently working on transitioning `request-promise-native` for all rdme commands to `node-fetch`. I'll have that in a separate PR!